### ISSUE TITLE
Assets 404

### ISSUE THREAD

**Issue by miklb** [bertrandkeller/overkyll-jekyll-theme#3]
Wanted to check out the framework, fired it up on a test install, all assets 404.

```
[2016-09-23 20:38:25] ERROR `/assets/js/instantclick.min.js' not found.
[2016-09-23 20:38:27] ERROR `/assets/js/instantclick.min.js' not found.
[2016-09-23 20:38:28] ERROR `/assets/favicons/favicon-32x32.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-180x180.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-152x152.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-144x144.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-120x120.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-114x114.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-76x76.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-180x180.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-72x72.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-152x152.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-60x60.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-144x144.png' not found.
[2016-09-23 20:38:31] ERROR `/assets/favicons/apple-icon-57x57.png' not found.
[2016-09-23 20:38:32] ERROR `/assets/favicons/apple-icon-120x120.png' not found.
[2016-09-23 20:38:32] ERROR `/apple-touch-icon-precomposed.png' not found.
[2016-09-23 20:38:32] ERROR `/assets/favicons/apple-icon-114x114.png' not found.
[2016-09-23 20:38:32] ERROR `/apple-touch-icon.png' not found.
[2016-09-23 20:38:32] ERROR `/assets/favicons/apple-icon-76x76.png' not found.
[2016-09-23 20:38:32] ERROR `/assets/favicons/android-chrome-192x192.png' not found.
```

As I understand it, Jekyll themes do not support asset folders yet. Unless I missed that documented in the readme, may want to point that out.

**Comment by bertrandkeller** [comment]
Thanks for the information. 
Jekyll 3.3 is coming, I'm testing how it is working now with the new feature.

**Comment by DirtyF** [comment]
Fixed in #5
